applications/ecallmgr/src/ecallmgr_fs_xml.erl

```diff
@@ -546,7 +546,6 @@ get_channel_vars({<<"Timeout">>, V}, Vars) ->
     case kz_term:to_integer(V) of
         TO when TO > 0 ->
             [<<"call_timeout=", (kz_term:to_binary(TO))/binary>>
-            ,<<"originate_timeout=", (kz_term:to_binary(TO))/binary>>
```

Comment:
@icehess why do you want to remove this ? isn't this going to affect the originate command timeout (not the ringing time) ?

Reply:
hmm, I'm trying to cancel `leg_timeout` in group confirm.